CHANGES.txt

```diff
@@ -22,14 +26,28 @@ Unreleased Changes (C++/Java/Python/PHP/Objective-C/C#/Ruby/JavaScript)
   * Optimize Descriptor.isExtensionNumber.
 
   Python
+  * [python-runtime] Add MethodDescriptor.CopyToProto() (#8327)
+  * Remove unused python_protobuf.{cc,h} (#8513)
+  * Start publishing python aarch64 manylinux wheels normally (#8530)
+  * Fix constness issue detected by MSVC standard conforming mode (#8568)
   * Make JSON parsing match C++ and Java when multiple fields from the same
     oneof are present and all but one is null.
 
+  Ruby
+  * Ruby: Add support for proto3 json_name in compiler and field definitions (#8356)
+  * Fixed memory leak of Ruby arena objects. (#8461)
+  * Fix source gem compilation (#8471)
+  * fix(ruby): Fix various exceptions in Ruby on 64-bit Windows (#8563)
```

Comment:
Maybe remove the `fix(ruby)` prefixes? I suppose this should be done on the branch first.